configuration.rst

```diff
@@ -58,7 +58,7 @@ Configuration Formats
 ~~~~~~~~~~~~~~~~~~~~~
 
 Unlike other frameworks, Symfony doesn't impose a specific format on you to
-configure your applications, but lets you choose between YAML, XML and PHP. 
```

Comment:
Can this be fixed in 5.4?

Reply:
Of course, here it is! https://github.com/symfony/symfony-docs/pull/17745